samples/server/petstore-security-test/php-slim/README.md

```diff
@@ -1,12 +1,58 @@
-# OpenAPI generated server
+# php-base - PHP Slim Server library for OpenAPI Petstore *_/ &#39; \&quot; &#x3D;end -- \\r\\n \\n \\r
```

Comment:
We may want to update `php-base` (packageName) with a better default value in another PR.

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java#L58

Reply:
@wing328 Got any suggestion for default value?